.github/workflows/release.yml

```diff
@@ -135,4 +135,48 @@ jobs:
           generateReleaseNotes: true
           token: ${{ secrets.GITHUB_TOKEN }}
           tag: ${{ github.event.inputs.releaseTag }}
-          prerelease: ${{ github.event.inputs.productionRelease && github.event.inputs.productionRelease == 'false' }}
\ No newline at end of file
+          prerelease: ${{ github.event.inputs.productionRelease && github.event.inputs.productionRelease == 'false' }}
```

Comment:
What change is this? I do not see any actual change. Please review the PR and only keep changes you intended to do.